app/helpers/application_helper/toolbar/physical_server_center.rb

```diff
@@ -1,5 +1,36 @@
 class ApplicationHelper::Toolbar::PhysicalServerCenter < ApplicationHelper::Toolbar::Basic
   button_group(
+    'physical_server_vmdb',
+    [
+      select(
+        :physical_server_vmdb_choice,
+        'fa fa-cog fa-lg',
+        t = N_('Configuration'),
+        t,
+        :items => [
+          button(
+            :physical_server_refresh,
+            'fa fa-refresh fa-lg',
+            N_('Refresh relationships and power states for all items related to the selected Physical Servers'),
+            N_('Refresh Relationships and Power States'),
+            :url_parms => "main_div",
+            :confirm   => N_("Refresh relationships and power states for all items related to the selected Physical Servers?"),
+            :enabled   => false,
```

Comment:
We need identical changes here as well.
No `:enabled` and `:onwhen` necessary

Reply:
@AparnaKarve, I think I got them all now.  Thanks for catching that.

Reply:
Thanks @rodneyhbrown7 . Looks good now.